### PR TITLE
feat: support additional GH org via NPM config

### PR DESCRIPTION
**Pull request by petermuessig**

### CODE REVIEW

**Comment by petermuessig** [comment]
*PLEASE DO NOT MERGE FOR NOW*!

This change has been created to enable the usage of the Easy UI5 Generator for additional GH organisations via command line parameter or via NPM configuration. 

`npm config set easy-ui5_addGhOrg XYZ`

@marcelschork - please validate the PR on your side whether it works for your additional GH org

**Comment by petermuessig** [comment]
@nicogeburek: the change has been validated by @marcelschork and his comment is in #89 

> Your implementation looked good to me 👍

I think we can go ahead and merge this and the other PR. Please also make a quick review of this change. Once this PR and #89 is merged we can also go for a new "minor" release `3.2.0` of Easy UI5.

**Review by nicogeburek — APPROVED** [review]
Looks good to me, thanks

**Pull request merged**
